src/ColorContextMenu-Core.package/ColorChooserMorph.class/methodProperties.json

```
@@ -2,14 +2,17 @@
 	"class" : {
 		 },
 	"instance" : {
+		"buildDiscreteMenu:" : "Kiwi 2/9/2021 14:33",
+		"buildDiscreteMenu:attribute:" : "Kiwi 2/9/2021 14:17",
+		"buildDiscreteMenu:attribute:levels:" : "Kiwi 2/9/2021 14:46",
```

```suggestion
		"buildDiscreteMenu:attribute:levels:" : "ct 2/8/2021 19:44",
```

Relates https://github.com/LinqLover/ColorContextMenu/pull/9/files#r574511174